gateway-getting-started-guide-turris-omnia.md

```diff
@@ -0,0 +1,79 @@
+# Getting Started
+
+## WebThings Gateway for Turris Omnia
+
+**Warning:** WebThings Gateway for Turris Omnia builds are currently highly experimental. So far the WebThings Gateway web interface only provides very basic router settings and it is not suitable for replacing your existing router software. We do not recommend using these builds for your primary home router.
+
+<img src="/images/turris_omnia.png" width="400">
+
+[WebThings Gateway](https://iot.mozilla.org/gateway/) is a software distribution for smart home gateways which allows users to directy monitor and control their smart home over the web, without a middleman.
+
+### What you will need
+
+1. **A [Turris Omnia](https://www.turris.cz/en/omnia/)** router
+2. **A USB stick** (at least 256MB)
+3. **USB dongles** (see the list of [compatible adapters](https://github.com/mozilla-iot/wiki/wiki/Supported-Hardware#adapters))
+
+**Note:** The Turris Omnia comes with Wi-Fi built in. The USB dongles are needed if you want to support other smart home protocols like Bluetooth, Zigbee and Z-Wave.
+
+### 1. Download Image
+
+First download the latest Turris Omnia gateway image from the [Mozilla IoT website](https://iot.mozilla.org/gateway/).
+
+### 2. Copy Image
+
+Next you will need to copy the downloaded .tar.gz file onto your USB stick. It should be the only file on the drive.
+
+### 3. Install Image
+
+With the router powered on:
+1. Insert the USB stick into one of the USB ports on the router
+2. Optionally plug any additional USB dongles into the other USB port
+3. Plug one end of a network cable into the WAN port on the back of the router, and the other into a LAN port on your modem or existing broadband router
+4. Press and hold the blue reset button on the back of the router until 4 LEDs (power, 0, 1 and 2) are lit 
+5. Release the reset button
+
+The router will show various sequences of lights as the image is copied and it reboots. It can take around 5 minutes for this to complete.
+
+### 4. Connect Wi-Fi
+When the router finishes starting up it will create a Wi-Fi hotspot called "**WebThings Gateway XXXX**" (where XXXX are four digits from your router's MAC address). Connect to that Wi-Fi hotspot using your desktop/laptop computer, smartphone or tablet.
+
+<img src="./images/wifi_ssid.png" width="300">
+
+Once connected you should be prompted to "create a new Wi-Fi network" by configuring  the network name (SSID) and password.
```

Comment:
nit: if I remember correctly the text "create a new Wi-Fi network" in the UI has "create" capitalized

There's also a double space after configuring

Reply:
Done.